content/pages/security.md

```diff
@@ -4,9 +4,21 @@ save_as: security.html
 template: security
 
 ## How to report a security issue
-If you believe you have discovered a vulnerability in Solr, you may first want to consult the [list of known false positives](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) to make sure you are reporting a real vulnerability.
-Then please disclose responsibly by following [these ASF guidelines](https://www.apache.org/security/) for reporting.
 
+### CVEs in Solr dependencies
+
+Solr depends on lots of other open-source software -- "dependencies".  If a CVE is published (a publicly identified vulnerability) against one of them, the Solr project will review it to see if it's actually exploitable in Solr -- usually they aren't.  Please review the [officially published non-exploitable vulnerabilities](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) before taking any steps.  If you **don't** see a CVE there, you should take the following steps:
```

Comment:
do as you wish, but I prefer that when writing markdown/asciidoc, that sentences start on a new line.  Much of our Ref Guide thankfully was upgraded to this style by Cassandra.